superset/common/query_context.py

```diff
@@ -24,11 +24,12 @@
 import pandas as pd
 from flask_babel import gettext as _
 
-from superset import app, cache, db, security_manager
+from superset import app, db
 from superset.common.query_object import QueryObject
 from superset.connectors.base.models import BaseDatasource
 from superset.connectors.connector_registry import ConnectorRegistry
 from superset.exceptions import QueryObjectValidationError
+from superset.extensions import cache_manager, security_manager
```

Comment:
Import the `cache_manager` instead of `cache` to make it clear which cache we are using.